### DISCUSSION TITLE
Unplayably Dark?

### DISCUSSION THREAD

**Discussion by garroh** [sultim-t/xash-rt#85] [Q&A]
So I just got this installed, and when starting a new game, much of the opening chapter is unlit, including textures that emit light in the original game. Anyone know a fix? I unzipped everything into the root folder a fresh HL install.

**Comment by ghost** [comment]
I'm having the same issue. I have a 6600 XT for reference.

**Reply by garroh** [reply]
I figured it out! I was only extracting the RT and RT-Bin folders into my Half Life directory. I needed to also extract the RT-Resources folder!

**Reply by ghost** [reply]
I also figured that out just now- whoops. Glad you got it.

**Reply by 2few** [reply]
i extracted both hl1-rt and hl1-rt-resources into the directory and my game is still extremely dark. even toggling to the original looks like something is wrong. the ray tracing is definitely working, but there's something wrong with the non raytraced lighting.
[sreenshot for comparison](https://i.imgur.com/qxwZICC.jpg) top is from youtube bottom is mine. almost looks like there is a lighting layer missing from mine. amd card also.
